StubProj/StubProj.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
StubProj is a workaround for getting C# debugging working in Visual Studio. See [https://gist.github.com/31/302bab2630ddf8d90ab4aec0d358b538](https://gist.github.com/31/302bab2630ddf8d90ab4aec0d358b538).